spec/values/calculation/abs.hrx

```diff
@@ -0,0 +1,158 @@
+<===> README.md
+Most of the same behavior tested for `calc()` applies to `abs()`, but for
+terseness' sake isn't tested explicitly.
+
+<===>
+================================================================================
+<===> zero/input.scss
+a {b: abs(0)}
+
+<===> zero/output.css
+a {
+  b: 0;
+}
+
+<===>
+================================================================================
+<===> positive/integer/input.scss
+a {b: abs(1)}
+
+<===> positive/integer/output.css
+a {
+  b: 1;
+}
+
+<===>
+================================================================================
+<===> positive/decimal/input.scss
+a {b: abs(5.6)}
+
+<===> positive/decimal/output.css
+a {
+  b: 5.6;
+}
+
+<===>
+================================================================================
+<===> negative/integer/input.scss
+a {b: abs(-17)}
+
+<===> negative/integer/output.css
+a {
+  b: 17;
+}
+
+<===>
+================================================================================
+<===> negative/decimal/input.scss
+a {b: abs(-123.456)}
+
+<===> negative/decimal/output.css
+a {
+  b: 123.456;
+}
+
+<===>
+================================================================================
+<===> named/input.scss
```

Comment:
This is testing a Sass variable as an argument, not a named argument (which wouldn't be a calculation anyway, so should be tested with the Sass function). `sass_variable` would be an appropriate name for this test.

Reply:
I changed this test to a named argument. :)